docs/dashboard/configuration/products.md

```diff
@@ -5,36 +5,53 @@ title: Product Types
 
 ## Product types
 
-Define the type of products you sell and organize them into groups. The Product Types list shows the type of product, a description of whether it is a physical or digital product, and which tax applies to its sale.
+In this page you can define the type of products you sell and organize them into groups. The Product Types list shows the type of product, an information if it is a physical or digital product, and which tax applies to its sale.
```

Comment:
"shows the type of product" - I think this would better describe rendering a product type as a column in the products table. Here we could simply say "shows the name of the product type".